src/components/02_atoms/PageTitle/PageTitle.js

```diff
@@ -32,7 +32,8 @@ const reducePropsToState = propsList => {
 };
 
 const handleStateChangeOnClient = title => {
-  document.title = title || '';
+  document.title =
```

Comment:
I'm wondering why do we allow an array as title in the first place? Shouldn't we disallow that?

Reply:
```

Reply:
Do you mind documenting this in the code?